lisa/features/disks.py

```diff
@@ -47,6 +47,9 @@ def get_raw_data_disks(self) -> List[str]:
     def get_all_disks(self) -> List[str]:
         raise NotImplementedError
 
+    def get_disk_controller_type(self) -> schema.DiskControllerType:
```

Comment:
Differentiate the name more like "get_hardware_disk_controller_type".

Reply:
Method changed to "get_hardware_disk_controller_type"